compiler/codegen/OMRCodeGenerator.cpp

```diff
@@ -285,123 +285,6 @@ OMR::CodeGenerator::initialize()
    cg->createObjectFormat();
    }
 
-
-OMR::CodeGenerator::CodeGenerator() :
```

Comment:
@0xdaryl I removed this snapshot branch specific constructor as I think the `master` changes removed the need for it....

Reply:
Yes, that's the correct thing to do.